jobs/ci-kubernetes-e2e-gce-canary.env

```diff
@@ -0,0 +1,4 @@
+# Everything else is the same as the other job it is testing
+BOOTSTRAP_METADATA_CANARY_IMAGE=true
```

Comment:
This is just metadata. `BOOTSTRAP_METADATA_FOO=bar` will result in adding a `{"foo": "bar"}` entry into the metadata.json dictionary.